# Reward Params
reward:
  give_final_reward: False
  max_final_reward: 20
  min_final_reward: -20
  max_step_reward: 0
  min_step_reward: -1.25

# Ambulance-Related Params
ambulance:
  amb_start_x: -50
  amb_goal_x: 50
  rel_amb_y_min: -41
  rel_amb_y_max: 16

# Enable Gazebo Gui Param
gazebo_gui: False
